redis-utils/src/main/java/org/ballerinalang/redis/actions/SortedSetActions.java

```diff
@@ -0,0 +1,386 @@
+/*
+ * Copyright (c) 2020, WSO2 Inc. (http:www.wso2.org) All Rights Reserved.
+ *
+ * WSO2 Inc. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http:www.apache.orglicensesLICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.ballerinalang.redis.actions;
+
+import org.ballerinalang.jvm.BallerinaErrors;
+import org.ballerinalang.jvm.values.HandleValue;
+import org.ballerinalang.jvm.values.api.BArray;
+import org.ballerinalang.jvm.values.api.BMap;
+import org.ballerinalang.redis.RedisDataSource;
+
+import static org.ballerinalang.redis.Constants.REDIS_EXCEPTION_OCCURRED;
+
+/**
+ * Redis sorted set actions.
+ */
+public class SortedSetActions extends AbstractRedisAction {
+
+    /**
+     * Add one or more members to a sorted set, or update its score if it already exist.
+     *
+     * @param redisDataSourceHandleValue redis datasource
+     * @param key The key of the sorted set
+     * @param memberScoreMap A map of members and corresponding scores
+     * @return The number of elements that were added to the sorted set, not including all the elements which were
+     *         already present in the set for which the score was updated
+     */
+    public static Object zAdd(HandleValue redisDataSourceHandleValue, String key, BMap memberScoreMap) {
+        try {
+            RedisDataSource redisDataSource = (RedisDataSource) redisDataSourceHandleValue.getValue();
+            return zAdd(key, redisDataSource, createMapFromBMap(memberScoreMap)).intValue();
+        } catch (Throwable e) {
+            return BallerinaErrors.createError(REDIS_EXCEPTION_OCCURRED, e.getMessage());
+        }
+    }
+
+    /**
+     * Get the number of members in a sorted set.
+     *
+     * @param redisDataSourceHandleValue redis datasource
+     * @param key The key of the sorted set
+     * @return The cardinality (number of elements) of the sorted set
+     */
+    public static Object zCard(HandleValue redisDataSourceHandleValue, String key) {
+        try {
+            RedisDataSource redisDataSource = (RedisDataSource) redisDataSourceHandleValue.getValue();
+            return zCard(key, redisDataSource).intValue();
```

Comment:
```suggestion
            return zCard(key, redisDataSource);
```
we can just pass the long value from zCard function